tests/test_order_LocalWl.py

```diff
@@ -116,15 +130,16 @@ def test_shape(self):
         positions = np.random.uniform(-box.Lx/2, box.Lx/2,
                                       size=(N, 3)).astype(np.float32)
 
-        comp = freud.order.LocalWlNear(box, 1.5, 6, 12)
+        comp = freud.order.LocalWlNear(box, 0.1, 6, 12)
         comp.compute(positions)
 
         npt.assert_equal(comp.Wl.shape[0], N)
 
     def test_identical_environments(self):
         (box, positions) = util.make_fcc(4, 4, 4)
 
-        comp = freud.order.LocalWlNear(box, 1.5, 6, 12)
+        # Use a really small rcut to ensure that it is used as a soft cutoff
```

Comment:
Good call! 👍 

Maybe add more of those comments to ensure that nobody changes that back at some point?